### DISCUSSION TITLE
New Camera Added but Not showing up

### DISCUSSION THREAD

**Discussion by gazunni** [mrlt8/docker-wyze-bridge#941] [General]
I added a new Wyze Camera today to my system.  The docker wyze bridge is showing that an extra camera has been added in the summary at the top, however, it is not showing the new camera view.  I have tried restarting the bridge, but it does not make a difference.   Any guidance would be appreciated.

**Comment by mrlt8** [comment]
Is it by any chance one of the newer models like the OG or Floodlight Pro? those aren't supported yet.

**Comment by gazunni** [comment]
It is the Wyze Cam Pan V3


________________________________

**Reply by mrlt8** [reply]
From the webUI, can you go to restart > clear cache and reconnect, then reload the page?

**Comment by gazunni** [comment]
While I now have the new Camera showing, none of the cameras will live stream.  All of the URL’s have switched from local IP addresses to some long string word such at the one below.
rtsp://zwtronecshswnbbyrseacmlgsn7li3
________________________________

**Reply by mrlt8** [reply]
hmm that is strange. can you post any customizations you have in your docker-compose.yml?

**Comment by gazunni** [comment]
No customizations.  The only files I have are shown in screen capture below.

[Image.jpeg.jpeg]

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
